app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find_by(id: params[:id])
+    @user = @post.author
+  end
+end
```

Comment:
- [ ] Please create an index action in the `Posts` controller and there's no need to add anything, the method can be empty.

Reply:
@thecodechaser  Hi! So, Do I just have to create an index and delete all the other methods?
Thanks !!